sdk/dotnet/Config/Config.cs

```diff
@@ -160,7 +158,7 @@ public static class Config
 
         public static string? ResourceManagerCustomEndpoint { get; set; } = __config.Get("resourceManagerCustomEndpoint");
 
-        public static string? ResourceManagerV2beta1CustomEndpoint { get; set; } = __config.Get("resourceManagerV2beta1CustomEndpoint");
+        public static string? ResourceManagerV2CustomEndpoint { get; set; } = __config.Get("resourceManagerV2CustomEndpoint");
```

Comment:
The renaming of the custom endpoint is because they have moved to GA rather than using the beta endpoint

I am ok with the fact that this would affect a super small number of users and we can note this on the CHANGELOG as it's only a config time parameter